docs/usage/remote-plugins.rst

```diff
@@ -45,3 +46,48 @@ but it can make asynchronous requests, i.e. passing ``async=True``.
 
 You need to run ``:UpdateRemotePlugins`` in Neovim for changes in the specifications to have effect.
 For details see ``:help remote-plugin`` in Neovim.
+
+If you are working on your plugin locally, it may be desirable to run the
+plugin from a git repository directly without the need of installing or
+symlinking it.
+The easiest way to do this is to create a dedicated vimrc file just for sake of
+development of your plugin. If you are in a root of the git repo with the
+plugin, consider creating this vimrc file:
+
+.. code-block:: console
+
+    $ cat vimrc
+    let &runtimepath.=','.escape(expand('<sfile>:p:h'), '\,')
+
+It will append your current work directory to the runtime path so neovim is
+able to find and load your plugin. You can now invoke neovim:
```

Comment:
"neovim" is spell "Nvim" in technical resources (i.e., almost everywhere except the website).

Reply:
Good catch. I did `s/neovim/Neovim/g` to match rest of the doc.